relentless/optimize/method.py

```diff
@@ -219,15 +226,15 @@ def find(self, objective, start, end):
         if targets[1] > 0 or np.abs(targets[1]) < self.abs_tol:
             result = end
         else:
-            steps = np.array([0, max_step])
+            steps = np.array([0,1], dtype=np.float64)
             iter_num = 0
             new_target = np.inf
             new_res = targets[1]
             while np.abs(new_target) >= self.abs_tol and iter_num < self.max_iter:
```

Comment:
I think we said we wanted to switch the stopping criterion for the `LineSearch` to use a relative tolerance. You can compute that tolerance once (outside the loop) as:
```py
tol = self.rel_tol*np.abs(targets[0])
```
See iii) of https://en.wikipedia.org/wiki/Wolfe_conditions

This would mean you need to change the `abs_tol` property throughout, as well.